src/components/city/BuildingItem.tsx

```
@@ -21,38 +18,15 @@ export default function BuildingItem({ category, level }: BuildingItemProps) {
-            image = health;
-            label = '운동';
-            building = '헬스장';
-            explain = '운동(헬스, 스포츠 등) 카테고리 관련 강의를 수강할 수 있는 페이지로 이동합니다.'
-            break;
-
         case 'beauty':
-            image = beauty;
-            label = '뷰티';
-            building = '백화점';
-            explain = '뷰티(패션, 화장 등) 카테고리 관련 강의를 수강할 수 있는 페이지로 이동합니다.'
+            const categoryMeta = getCategoryMeta(category as CategoryUrl);
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)